modules/Package.swift

```diff
@@ -66,7 +66,7 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "1.1.0"),
         .package(url: "https://github.com/pointfreeco/swift-url-routing", from: "0.6.0"),
         .package(url: "https://github.com/zcash-hackworks/MnemonicSwift", from: "2.2.4"),
-        .package(url: "https://github.com/zcash/ZcashLightClientKit", from: "2.0.8"),
+        .package(url: "https://github.com/zcash/ZcashLightClientKit", from: "2.0.9"),
```

Comment:
Non-blocking: we should update this to `https://github.com/Electric-Coin-Company/zcash-swift-wallet-sdk` at some point.